src/handlers/update-id.ts

```diff
@@ -0,0 +1,11 @@
+import { RequestHandler } from 'express'
+
+import { logger } from '../config/logger'
+
+export const updateId: RequestHandler = (req, res) => {
+  logger.info('updateId')
+
+  const { id } = req.params
+
+  res.status(200).send(`${id} has been updated`)
```

Comment:
Use http-status-codes library
```suggestion
  res.status(StatusCodes.OK).send(`${id} has been updated`)
```